.github/workflows/e2e-test.yaml

```diff
@@ -28,8 +28,8 @@ jobs:
         env:
           # kubectl is supported within one minor version (older or newer) of k8s
           # https://kubernetes.io/releases/version-skew-policy/#kubectl
-          KUBECTL_VERSION: v1.26.5
```

Comment:
I would double-check why this was 1.26 in the first place (maybe @bayandin had a reason for it?), but otherwise looks good

Reply:
I don't see any kubectl version bindings in e2e tests or anything like that.

Reply:
Ah, I mean because that kubectl version was changed from v1.25.9 -> v1.26.5 by #246

Reply:
> I would double-check why this was 1.26 in the first place (maybe @bayandin had a reason for it?), but otherwise looks good

There's no particular reason, it just should work, according to the comment above.
Should be ok to change it to a matching k8s version 😄